README.md

```diff
@@ -33,113 +32,60 @@ Please do not copy and paste the commands we provide in this readme without :
-
-### Running the bot on your computer. (windows)
-
-- Install git : https://git-scm.com/download/win
-- Download the bot : `git clone https://github.com/BastLast/DraftBot-A-Discord-Adventure`
-- Install NodeJS http://nodejs.org/fr/ 
-  - Check package.json for the minimal required version
-  - Our docker image is based on node alpine (see the DockerFile for the exact version)
-  - We recommend that you use nvm to manage your node versions If you use nvm, you can use `nvm install` to install the correct version If you don't have nvm already install, you can use `npm install -g nvm` to install it
-- Install Yarn `npm install --global yarn`
-- Install the bot : `yarn install`
-- Install and launch a mariadb database. Keep the credentials for the config file. We provide a tutorial below for this step.
-- Create the config file by copying the file config.default.toml in a new file config.toml
-- Edit the file config.toml with the correct data
-- Launch the bot : `yarn start`
-
-### Here is the same guide for debian based linux distributions (or WSL and git bash on windows)
-
-- Install git : `apt-get install git`
-- Download the bot : `git clone https://github.com/BastLast/DraftBot-A-Discord-Adventure`
-- Install NodeJS `apt-get install nodejs`
-  - Install npm `apt-get install npm`
-  - Check package.json for the minimal required version
-  - Our docker image is based on node alpine (see the DockerFile for the exact version)
-  - We recommend that you use nvm to manage your node versions If you use nvm, you can use `nvm install` to install the correct version If you don't have nvm already install, you can use `npm install -g nvm` to install it
-- Install yarn `npm install -g yarn` (you may have to install npm and use sudo)
-- Install the bot : `yarn install`
-- Install and launch a mariadb database. Keep the credentials for the config file. We provide a tutorial below for this step.
-- Create the config file : `cp config/config.default.toml config/config.toml`
-- Edit the file config.toml with the correct data
-- Launch the bot : `yarn start`
-
-### Updating the bot
-
-- Be sure to use `yarn install` and `yarn start` each time you update the bot.
-- Migrations will run automatically, but be sure to check them as they are created for our database, they may cause
-  issues in yours.
-
-Make sure to have docker installed on your machine.
-Here is a link to the windows installation guide : https://docs.docker.com/desktop/install/windows-install/
-Here is a link to the linux installation guide : https://docs.docker.com/engine/install/ubuntu/
-
-Please follow the "without docker" steps until the `yarn install` step (you don't need to do it).
+## BETA BRANCH WARNING
 
-### Compile the docker image
+You are here in the not yet released v5 of DraftBot. Compared to previous versions of the game, we migrated to microservices. This cloning this repository will allow you to get all the services necessary to run draftbot v5.
```

Comment:
"**This** cloning this repository (...)"... Le premier "this" de cette phrase devait probablement être un "th**u**s", je suppose.